app/views/partials/foyer/individu-form.html

```diff
@@ -184,21 +184,24 @@
     </div>
   </div>
 
-  <div class="form-group from-group-lg" ng-if="captureTauxInvalidite()">
+  <div class="form-group from-group-lg" ng-if="capturetauxIncapacite()">
     <label class="control-label col-sm-4" for="eligibiliteAss">
-      Quel est {{ isDemandeur() ? 'votre' : 'son' }} taux d'invalidité ?
+      Quel est {{ isDemandeur() ? 'votre' : 'son' }} taux d'incapacité ?
+      <div class="small">
```

Comment:
Mmh, ok. It looks like `small` is for help text on the left handside, `help-block` on the right handside. Meh.

> Most consistent IMO: get rid of the “Conditions particulières” help text, it doesn't add anything, and make the taux help text a right handside `help-block`, like everywhere else in the app.